engine/classes/Elgg/Access.php

```diff
@@ -1,74 +1,14 @@
 <?php
 namespace Elgg;
+
 /**
- * Class used to determine if access is being ignored.
+ * Class formerly used to determine if access is being ignored.
  *
- * @package    Elgg.Core
- * @subpackage Access
- * @access     private
- * @see        elgg_get_ignore_access()
+ * This exists now purely to be extended by ElggSession, so that elgg_get_access_object()
+ * can return the session while being an instance of this.
  *
- * @todo       I don't remember why this was required beyond scope concerns.
+ * @access private
+ * @see    \ElggSession
  */
 class Access {
-	/**
-	 * Bypass Elgg's access control if true.
-	 * @var bool
-	 */
-	private $ignore_access;
-
-	// @codingStandardsIgnoreStart
-	/**
-	 * Get current ignore access setting.
-	 *
-	 * @return bool
-	 * @deprecated 1.8 Use \Elgg\Access::getIgnoreAccess()
-	 */
-	public function get_ignore_access() {
-		elgg_deprecated_notice('\Elgg\Access::get_ignore_access() is deprecated by \Elgg\Access::getIgnoreAccess()', 1.8);
```

Comment:
We broke this by changing the class name in 1.10. Do we care about keeping this?